20181004/AGENDA.md

```diff
@@ -0,0 +1,14 @@
+# Sardana Follow-up Meeting - 2018/10/04
+
+To be held on 2018/10/04 at 14:00  
+This one will be a bit unusual, since we will focus more on improving our work
+
+# Agenda
+1. Quick overview of urgent issues
+	* Alternative to appear.in
+	* Current development problems
+	* New urgent issues (if any)
+2. Brainstorming
```

Comment:
Thanks for posting the brainstorming ideas. I let myself to add more.. but it is not mandatory to add yours now. We will have time on the brainstorming to catch them...
First of all I see like two categories here. First are the objectives or topics to improve, feedback on.. and then are the tools. So propose to talk about the following objectives:
* follow-up meetings
* issues priorization/triage
* colaborative development
* communication with users
* releases

And the tools that could be used to improve these objectives:
* video conferences tools
* teams discussions
* team collaborative tools e.g. slack
* project boards
* issues labels
* milestones
* newsletter
* questioners